policyengine_us/variables/gov/states/tax/income/state_itemized_deductions.py

```diff
@@ -8,3 +8,70 @@ class state_itemized_deductions(Variable):
     unit = USD
     definition_period = YEAR
     adds = "gov.states.household.state_itemized_deductions"
+
+    def formula(tax_unit, period, parameters):
+        # States that adopt the federal itemized deductions
+        # Based on comments in state_itemized_deductions.yaml
+        FEDERAL_ITEMIZED_DEDUCTION_STATES = [
+            "CT",  # Connecticut
+            "GA",  # Georgia
+            "ND",  # North Dakota
+            "SC",  # South Carolina
+            "UT",  # Utah
+        ]
+
+        # Get the current state
+        state_code = tax_unit.household("state_code_str", period)
+
+        # Get the sum of state-specific itemized deductions
+        state_specific_base = add(
+            tax_unit,
+            period,
+            parameters(period).gov.states.household.state_itemized_deductions,
+        )
+
+        # Special handling for states that need individual vs joint itemization maximum
+        STATES_WITH_INDIVIDUAL_JOINT_MAXIMUM = {
+            "MT": {
+                "indiv": "mt_itemized_deductions_for_federal_itemization_indiv",
+                "joint": "mt_itemized_deductions_for_federal_itemization_joint",
+            },
+            "IA": {
+                "indiv": "ia_itemized_deductions_indiv",
+                "joint": "ia_itemized_deductions_joint",
+            },
+            "DE": {
+                "indiv": "de_itemized_deductions_indv",
+                "joint": "de_itemized_deductions_joint",
+            },
+            "AR": {
+                "indiv": "ar_itemized_deductions_indiv",
+                "joint": "ar_itemized_deductions_joint",
+            },
+        }
+
+        # Calculate maximum itemized deductions for applicable states
+        state_specific_with_max = state_specific_base
```

Comment:
clean up this and line 64, shouldn't need reassignment. same with state standard